.travis.yml

```diff
@@ -7,6 +7,10 @@ matrix:
       smalltalk: Squeak-trunk
 
     - os: linux
+      language: smalltalk
+      smalltalk: Squeak-5.0
+
+    - os: linux
```

Comment:
Yes. Since we no longer include the complete system package, this has been made possible by this PR.